### DISCUSSION TITLE
RESOLVED [March 3, 2023] GitHub Actions: Investigating reports of degraded performance

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#49104] [Actions]
**Posted 19:09 UTC - March 3, 2023** 
## We are aware of reports of degraded performance with GitHub Actions. 
- Please follow this GitHub Status report: [Investigating reports of degraded performance](https://www.githubstatus.com/incidents/sr7zptjd0b07) for more information
- You can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.

We appreciate your patience.

**Comment by Akash1134** [comment]
The incident was resolved on Mar 03, 2023 - 21:14 UTC roughly 3.5 hours after being reported. Please note that you are not being billed for these runs and new runs are not affected by the incident. 

For more details, see the GitHub Status report: [Investigating reports of degraded performance](https://www.githubstatus.com/incidents/sr7zptjd0b07). Thank you for your support and patience.
